config/database.yml

```diff
@@ -46,7 +46,9 @@ development:
 test:
   <<: *default
   database: education_test
-
+  host:     localhost
```

Comment:
This shouldn't have been committed, it's probably what's breaking the Travis testsuite.